crates/serpen/tests/fixtures/comprehensive_ast_rewrite/main.py

```
@@ -0,0 +1,103 @@
+
+
+def main():
+    """Main function demonstrating all the conflicts in action"""
+    # Using imported functions with aliases
+    db_result = db_process("database_data")
+    util_result = process("utility_data")  # Imported function
+    auth_result = auth_process("auth_data")
+
+    # Using imported classes with conflicts
+    util_logger = UtilLogger("util")
+    model_logger = Logger("model")  # Different Logger class
```

Pylint (3.3.7)

[error] Logger is not callable

(E1102)